com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameObjectBuilder.java

```diff
@@ -91,14 +92,15 @@ private PrototypeDesc.Builder loadPrototype(IResource input) throws IOException,
     }
 
     @Override
-    public Task<Void> create(IResource input) throws IOException, CompileExceptionError {
-        PrototypeDesc.Builder builder = loadPrototype(input);
-        TaskBuilder<Void> taskBuilder = Task.<Void>newBuilder(this)
+    public Task create(IResource input) throws IOException, CompileExceptionError {
+        TaskBuilder taskBuilder = Task.<Void>newBuilder(this)
                 .setName(params.name())
                 .addInput(input)
                 .addOutput(input.changeExt(params.outExt()))
                 .addOutput(input.changeExt(ComponentsCounter.EXT_GO));
 
+        PrototypeDesc.Builder builder = loadPrototype(input);
```

Comment:
One of places where having of an input proto message cache is especially useful. 
Here we not only get data but modify it. So before this fix we did it twice in create and in build. Now it's not necessary and use `loadPrototype` only once.